### PR TITLE
Add aggregated_properties parameter on assets.list()

### PR DESCRIPTION
**Pull request by sanderland**

### CODE REVIEW

File: tests/tests_unit/test_api/test_assets.py [hunk 1]
```diff
@@ -109,6 +109,14 @@ def test_list(self, mock_assets_response):
         assert "bla" == jsgz_load(mock_assets_response.calls[0].request.body)["filter"]["name"]
         assert mock_assets_response.calls[0].response.json()["items"] == res.dump(camel_case=True)
 
+    def test_list_ag(self, mock_assets_response):
```
Comment (erlendvollset) [inline tests/tests_unit/test_api/test_assets.py]: It's better if test names have more descriptive names, so something like `test_list_with_aggregated_properties_param`. It makes it easier to understand what has gone wrong just by looking at the test suite if the test should ever fail. Doesn't really matter if these method names become overly long

File: tests/tests_unit/test_api/test_assets.py [hunk 2]
```diff
@@ -109,6 +109,14 @@ def test_list(self, mock_assets_response):
...
+        res = ASSETS_API.list(name="bla",aggregated_properties=["childCount"])
+        assert ["childCount"] == jsgz_load(mock_assets_response.calls[0].request.body)["aggregatedProperties"]
+
+    def test_list_ag_camel(self, mock_assets_response):
```
Comment (erlendvollset) [inline tests/tests_unit/test_api/test_assets.py]: `test_list_with_aggregated_properties_param_when_snake_cased` instead maybe?

File: tests/tests_unit/test_api/test_assets.py (hunk 1 above)
Comment (sanderland) [reply tests/tests_unit/test_api/test_assets.py]: fixed

File: tests/tests_unit/test_api/test_assets.py (hunk 2 above)
Comment (sanderland) [reply tests/tests_unit/test_api/test_assets.py]: fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,10 +21,14 @@ Changes are grouped as follows
...
+### Added
+- Support for aggregated properties of assets.
+- `Asset` and `AssetList` classes now have a `sequences` function which retrieves related sequences.
+
 ### Changed
 - `list` and `__call__` methods for assets now support list parameters for `root_ids`, `root_external_ids`.
-- `Asset` class now has an `aggregates` property.
-- `Asset` and `AssetList` classes now have a `sequences` function which retrieves related sequences.
+
+### Changed
```
Comment (erlendvollset) [inline CHANGELOG.md]: Two `Changed` sections

Comment (sanderland) [reply CHANGELOG.md]: fixed

File: tests/tests_integration/test_api/test_assets.py [hunk 4]
```diff
@@ -86,6 +86,12 @@ def test_list(self, post_spy):
         assert 20 == len(res)
         assert 2 == COGNITE_CLIENT.assets._post.call_count
 
+    def test_list_ag(self, post_spy):
```
Comment (erlendvollset) [inline tests/tests_integration/test_api/test_assets.py]:
```suggestion
    def test_list_with_aggregated_properties_param(self, post_spy):
```

**Pull request merged**